src/map.js

```diff
@@ -865,12 +853,13 @@ var map = function (arg) {
    * Get, set, or create and set a file reader to a layer in the map to be used
    * as a drop target.
    *
-   * @param {string|object|undefined} readerOrName: undefined to get the
-   *    current reader, an instance of a file reader to set the reader, or a
-   *    name to create a file reader (see utils.createFileReader for options).
-   * @param {object} opts: options for creating a file reader.  If this
-   *    includes layer, use that layer, otherwise create a layer using these
-   *    options.
+   * @param {string|object} [readerOrName]: undefined to get the current
+   *    reader, an instance of a file reader to set the reader, or a name to
+   *    create a file reader.
+   * @param {object} [opts]: options for creating a file reader when the reader
```

Comment:
It doesn't seem to change the output here, but for consistency it would be good to remove the `:` in these declarations.